### PR TITLE
updated index.html & contributors.html

### PR DESCRIPTION
**Pull request by vyaspranjal33**
removed "4 update Readme.md" , fixed duplicate entries

### CODE REVIEW

File: contributors.html [hunk 1]
```diff
@@ -374,9 +374,17 @@ <h1 class="animated rubberBand delay-4s">Contributors</h1>
         <a class="box-item different-color" href="https://github.com/JustCaptcha">Albert Hofmann</a>
         <a class="box-item" href="https://github.com/katrinasilver">Katrina Agustin</a>
         <a class="box-item" href="https://github.com/sudouser89">Pavan kumar</a>
-	      <a class="box-item" href="https://github.com/AlexanderMalmstrom">Alexander Malmström</a>
+        <a class="box-item" href="https://github.com/AlexanderMalmstrom">Alexander Malmström</a>
+    	   <a class="box-item" href="https://github.com/BunditSpace">Bundit Wisedphanit</a>
+<<<<<<< HEAD
```
Comment (BennyCarlsson) [inline contributors.html]: looks like you forgot something here :)

**Comment by BennyCarlsson** [comment]
If you have time to fix the comment I left and the merge conflict we happily merge your pull request :)

**Comment by vyaspranjal33** [comment]
@BennyCarlsson  Done

**Comment by BennyCarlsson** [comment]
Awesome :)

**Pull request merged**